data/cities/detroit/corporations/iris.md

```diff
@@ -0,0 +1,37 @@
+name: Iris
+markets:
+    immobilier: 3
+    alimentaire: 3
+    cyberware: 1
+    santé: 2
+datasteal: 20
+sabotage: 20
+extraction: 10
+detection: 30
+on_first:
+    update('prometheus', 1)
+    update(ladder[1], 1)
+on_last:
+    update(ladder[0], 2)
+on_crash:
+    # TBD
+
+> *From the stars to you*
+
+Iris est la corporation la plus visible au quotidien. Détentrice de plus de 3 500 chaînes de télévision, propriétaire de multiples réseaux sociaux, Iris est la seule corporation à avoir installé son siège en orbite, déployant depuis l’espace son réseau de satellites de télécommunication afin d’abreuver la planète de divertissements, de news et de publicités en tout genre. 
+
+
+Iris a l’image d’une corporation bon enfant, en perpétuel renouvellement, au service de la jeunesse et de l’opulence. Elle inonde d’informations et flatte ses auditeurs. Elle organise des concours, des voyages touristiques dans l’espace, la télé-réalité, des galas, des cérémonies de remise de prix. Elle choisit les stars de demain et enterre celle d’hier. 
+
+
+Anarchiste, Conformiste, Conservateur, Révolutionnaire… Qui que vous soyez Iris a l’éventail de divertissement qu’il vous faut et vous vend le comprimé énergétique amincissant, le NewBotox, la crème anti-âge, le smartphone et le viagra 2.0 qui vous feront ressembler à votre acteur préféré. Ce contact direct avec le consommateur et le positionnement stratégique et emblématique de leur QG font d’Iris une corporation puissante qui a l’oreille de chaque foyer.
+
+
+---
+
+
+>Ah Iris, j’ai trouvé que c’était un joli petit nom pour Big Brother…  Du haut de leur station spatiale, ils balancent leurs merde, leurs émissions de cuisine, leurs jeux télévisés, leurs programmes pour adulte et leurs derniers iPear. Je pense que s’il y a si peu de gens dans les ombres, qui se rendent compte de la pourriture corporatiste, c’est en partie grâce à eux. Putain, ils sont partout, ils t’assomment… 
+
+>Si tu bosses pour eux, tu vas avoir le profil détaillé de chaque garde d’en face, leur feed sur les réseaux sociaux et la moindre de leurs habitudes… Le Net, une mine d’informations qui valent de l’or et c’est leur putain de terrain. C’est clair, si tu bosses bien, tu pourrais même devenir une star dans leur prochain film d’action. Par contre, il faut que tu le sache, ils ont la même fiche sur toi… Et les moyens de te faire payer… 
```

Comment:
ils ont la même fiche sur toi… Et les moyens de te faire payer… => ils ont la même fiche sur to, et les moyens de te faire payer. (points de suspension * 2)